org/pr/installable-build.ts

```diff
@@ -140,7 +140,9 @@ async function getDownloadCommentText(status) {
 export default async (status: Status) => {
     if (status.state == "pending" && HOLD_CONTEXTS.concat(HOLD_CONTEXTS_NO_COMMENT).includes(status.context)) {
       await markStatusAsSuccess(status)
-      await createOrUpdateComment(status, `You can trigger an installable build for these changes by visiting CircleCI [here](${status.target_url}).`)
+      if (HOLD_CONTEXTS.includes(status.context)) {
```

Comment:
Wouldn't it be more exact and clearer to write it like this? 🤔 

```suggestion
      if (!(HOLD_CONTEXTS_NO_COMMENT.includes(status.context))) {
```

Reply:
I see 🤔 I'm not sure this is clearer or more exact. I mean: it makes sense for us because we know that, historically, there was a single flow and we are setting up an exception. 

But, in general, if you don't know or don't remember the history, you have two functions and you want to call the second one only for a smaller set of contexts, so the negated condition may end up being less clear.

Reply:
Right. My reasoning when I read the code was more about "since this guards the call to `createOrUpdateComment`, this makes sense to me that we'd only do it for context we specified by the "…_NO_COMMENT" constant." But I thought this also probably because the `HOLD_CONTEXTS` constant name doesn't tell anything about comment or not, while `HOLD_CONTEXTS_NO_COMMENT` does have the "comment" condition in its name 🤷 

Alternatively, renaming the first constant `HOLD_CONTEXTS_COMMENT` (and then use your positive condition `if (HOLD_CONTEXTS_COMMENT.includes(status.context))` would have the same effect (and might even be better to make it clear about the differences? )

Anyway, I'm ok with both tbh, it's not a big deal and just stylistic anyway, so I'll leave it up to you to chose what feels clearer 🙂

Reply:
Makes sense. I updated the name of the arrays.